web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -43,6 +44,7 @@
     public CustomObjectMapper() {
 
         super.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        super.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
```

Comment:
good. But since the names of the enum values are used (if I understand correctly), we should make a note in the enum that any change to the enum names will be an alteration in the output of the REST controller. [to prevent unintended API alteration by someone who thinks they can improve the enum constant names].